pubspec.yaml

```diff
@@ -1,11 +1,8 @@
 name: pedantic
-version: 1.9.1
+version: 1.9.2
 description: >-
   The Dart analyzer settings and best practices used internally at Google.
 homepage: https://github.com/dart-lang/pedantic
 
 environment:
   sdk: '>=2.1.1 <3.0.0'
```

Comment:
We need to leave a dependency in here or publish this + the new meta at exactly the same time. The dependency would be on the new version of meta that _does not_ contain `unawaited`.

Reply:
My understanding is that you're trying to prevent the case where a version of `pedantic` that defined `unawaited` and a version of `meta` that also defines `pedantic` can both be selected by the version solver. If that's the case, then I think we have to include a dependency. Even if we publish them both at the same time a user could have `meta` pinned to a version that defines `unawaited` and the version solver could still pick a version of `pedantic` that also defines it.

Reply:
Correct, adding the dependency is the safest thing to do.

I think removing the dependency eventually is possible, it is much more common to pin `pedantic` than `meta` (people pin it so that their lints don't change, when they aren't aware of the special analysis_options.yaml files it provides for this purpose).